scipy/stats/unuran/unuran_wrapper.pyi

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import TYPE_CHECKING, Union, Any, Callable, Tuple, Optional, List
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
```

Comment:
`TYPE_CHECKING` checks are redundant inside `.pyi` stub files, as its value will always be `True` in there (no exceptions).

Reply:
> `TYPE_CHECKING` checks are redundant inside `.pyi` stub files, as its value will always be `True` in there (no exceptions).

Thanks for letting me know. I have removed this now.